tools/ktx/command_create.cpp

```diff
@@ -141,6 +143,12 @@ struct OptionsCreate {
                     " This option is mutually exclusive with --runtime-mipmap and cannot be used with UINT or 3D textures.")
                 (kEncode, "Encode the created KTX file. Case insensitive."
                     "\nPossible options are: basis-lz | uastc", cxxopts::value<std::string>(), "<codec>")
+                (kNormalize, "Normalize input normals to have a unit length. Only valid for\n"
+                    "linear normal textures with 2 or more components. For 2-component inputs\n"
+                    "2D unit normals are calculated. Do not use these 2D unit normals\n"
+                    "to generate X+Y normals for --normal-mode. For 4-component inputs\n"
```

Comment:
I think "with --normal-mode" is more accurate. Isn't it normal-mode that makes the X+Y normals?

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/977/commits/f1b94c8fd84923b76dcee2c080d9b5735827fb37